pkg/windows/service/service.go

```diff
@@ -69,7 +69,7 @@ func (h *handler) Execute(_ []string, r <-chan svc.ChangeRequest, s chan<- svc.S
 	h.fromsvc <- nil
 
 	s <- svc.Status{State: svc.Running, Accepts: svc.AcceptStop | svc.AcceptShutdown | svc.Accepted(windows.SERVICE_ACCEPT_PARAMCHANGE)}
-	klog.Infof("Service running")
+	klog.InfoS("Service running")
```

Comment:
This was just moving directly, but it would be valuable to have the `servicename` here.